omero/developers/Java.txt

```diff
@@ -253,6 +253,8 @@ This is useful when you need for example the pixels intensity.
         }
     }
 
+    rdf.close();
```

Comment:
you will have to remove that
as @dominikl pointed out, this method is not yet available
see https://trello.com/c/Uavp5lOO/30-gateway-facility